library/templates/address_list_display.php

```diff
@@ -15,76 +15,159 @@
 
 use OpenEMR\Services\ContactService;
 
-$table_id = uniqid("table_addresses");
+$table_id = uniqid("table_text_addresses_");
 
 $contactService = new ContactService();
 $addresses = $contactService->getContactsForPatient($_SESSION['pid']) ?? [];
+
+$list_address_types = generate_list_map("address-types");
+$list_address_uses = generate_list_map("address-uses");
 ?>
+
 <div id="<?php echo attr($table_id); ?>">
-    <template class="template-text-addresses">
-        <tr>
-            <td class='text-addresses-full-address'></td>
-        </tr>
+    <div class="table_text_addresses">
+        <div class ="text_addresses header label_custom row">
+            <?php echo xlt("Additional Addresses"); ?>
+        </div>
+        <div class="d-none no_addresses row">
+                <span class="text data"><?php echo xlt("NONE"); ?></span>
+        </div>
```

Comment:
nice to see the table stuff gone. However, since view is one of the last vestige of using tables in engine, I'd recommend we start refactoring to bootstrap and divs. Then use BS col grid same as edit that Rod had done. Meanwhile breaking datatypes out as separate twig templates.

Reply:
@sjpadgett
Sounds nice.  Refactoring code from table to div is probably well beyond the scope of what I am doing here.  

Not sure about twig templates if people won't use twig feature (i.e. calendar code is supposed to be twig templates but there is a lot of php code inserted in it using few twig templating feature.